warehouse/tuf/tasks.py

```diff
@@ -0,0 +1,64 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import redis
+
+from warehouse.tasks import task
+from warehouse.tuf.constants import TUF_REPO_LOCK
+from warehouse.tuf.services import IRepositoryService
+
+
+@task(bind=True, ignore_result=True, acks_late=True)
+def bump_snapshot(task, request):
+    """
+    Re-signs the TUF snapshot role, incrementing its version and renewing its
+    expiration period.
+
+    Bumping the snapshot transitively bumps the timestamp role.
+    """
```

Comment:
Minor nit:  This is the only docstring in this module, and the tasks all call into service methods, which are well documented. Can we just remove this docstring here?